src/kernels/batchnorm_functions.h

```diff
@@ -54,6 +59,31 @@
 
 #endif
 
+#if MIOPEN_USE_BFPMIX == 1
+#define _FLOAT ushort
+#ifdef _FLOAT_PREC
+#undef _FLOAT_PREC
+#endif
+#define _FLOAT_PREC float
+
+#ifdef EPSILON
+#undef EPSILON
+#endif
+#define EPSILON (_FLOAT)0.00781250
```

Comment:
`EPSILON` expands to `(ushort)0.00781250` which means 0

Reply:
Oh interesting.
I don't think it's actually used in this case.